cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -1189,6 +1205,11 @@ private int countNumberOfBranchesUsingParameter(SwitchInstruction switch, Parame
  * int x = *p;
  * ```
  * does not preserve the identity of `*p`.
+ *
+ * Similarly, functions that copy the contents of a string into a new location
+ * does not also preserve the identity. For example, `strdup(p)` does not
+ * preserve the identity of `*p` (since it allocates new storage and copies
+ * the string into the new storage).
```

Comment:
```suggestion
 * Similarly, a function that copies the contents of a string into a new location
 * does not also preserve the identity. For example, `strdup(p)` does not
 * preserve the identity of `*p` (since it allocates new storage and copies
 * the string into the new storage).
```